### ISSUE TITLE
talk about hubs/code hosts generally, mention alternatives to GitHub

### ISSUE THREAD

**Issue by NickleDave** [ResearchCodeReviewCommunity/dev-review#15]
we need to make sure to mention alternatives to hosting code for review
e.g., GitLab

**Comment by tlestang** [comment]
Yeah this is something we should discuss. Actually.. in my opinion there is no need of hosting code since code reviews are live, i.e author and reviewers sit together and share a laptop (or their screen in the case of a call).

But hosting code might come in handy in some cases?
